lib/js/src/rpc/RpcMessage.js

```diff
@@ -54,32 +54,32 @@ class RpcMessage extends RpcStruct {
     /**
      * Initializes an instance of RpcMessage.
      * @class
-     * @param {Object} store - A config object containing attributes such as rpcType, functionName, correlationID, bulkData, and parameters.
+     * @param {Object} store - A config object containing attributes such as messageType, functionName, correlationID, bulkData, and parameters.
```

Comment:
This doc change is not made in the RpcNotification, Request, and Response classes. It still says rpcType in those files.